src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -296,6 +303,92 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
             return default;
         }
 
+        private static ImmutableArray<TaggedText> GetConstantValueContent(
+            Workspace workspace,
+            SyntaxToken token,
+            ISyntaxFactsService syntaxFacts,
+            SemanticModel semanticModel,
+            CancellationToken cancellationToken)
+        {
+            if (!syntaxFacts.IsBinaryExpression(token.Parent))
+            {
+                return default;
+            }
```

Comment:
why not any expression that can be constant?

Reply:
Because the issue asked for just binary expressions. I would prefer to have this on any expression too, I'll wait for the design review though.

Reply:
i'd say that things don't need to be terribly literal (no pun intended) :)

Reply:
@CyrusNajmabadi I asked what kinds of other expressions this should show up in with an extensive comment below, and you said none of them. So it's unclear to me what you were requesting here.

Reply:
i'm ok with this behavior.